modules/oe_whitelabel_helper/config/post_updates/00002_date_range_formats/daterange_compact.format.oe_whitelabel_date_time_long.yml

```
@@ -0,0 +1,16 @@
+langcode: en
+status: true
+dependencies: {  }
+id: oe_whitelabel_date_time_long
+label: 'Date and time, long names'
+default_pattern: 'l d F Y, H.i (T)'
+default_separator: ' - '
+same_day_start_pattern: 'l d F Y, H.i'
+same_day_end_pattern: 'H.i (T)'
+same_day_separator: ' - '
```

```suggestion
same_day_separator: '-'
```
Align with config/install.
According to the pdf there are no spaces on same day.